### ISSUE TITLE
not working properly with a dark theme

### ISSUE THREAD

**Issue by timoc** [d1vanov/quentier#221]
When using it with a drak theme the 'note' viewer uses the default background color which in my case is black. This does not work with the note editor, as the text is black on a black background. Some mechanism to set a different background is needed.

**Comment by d1vanov** [comment]
I'm quite surprised the note editor picks black background color - it's actually a web page rendered by either QtWebKit or QtWebEngine depending on the version of Quentier.

Could you please specify your distro and theme name? Also, could you please post a screenshot of how it looks like?

**Comment by timoc** [comment]
Os Version:
```
PRETTY_NAME="Ubuntu 18.04.1 LTS"
VERSION_ID="18.04"
```
but Kubuntu spin.
```
theme: zvon back

![image](https://user-images.githubusercontent.com/140698/44748509-301f8a80-ab10-11e8-9cea-0e5431c05df4.png)

**Comment by d1vanov** [comment]
Pushed the fix for this one to both master and development branches. Also removed one old hack which, as it turned out, was capable of producing out of memory crash + backported to master from development the fix for tags loading on open note editor tabs on startup.

Here's how Quentier looks now with the dark theme:
![quentier_dark_theme](https://user-images.githubusercontent.com/2050562/44954362-7591de00-aea9-11e8-8ac7-b547f847c51a.png)
So basically what I did is enforcing of white background color for note editor. In future I'll think about the possibility to support dark theme for the note editor as well - that would require reverting the color of text (from default black to white) and it's somewhat complicated and not easy to do properly. So for now such minimal fix should suffice.
